build_tools/integrate/llvm_bump_revision.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# Copyright 2024 The StableHLO Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SCRIPT_DIR="$(dirname "$(realpath "$0")")"
```

Comment:
I noticed that stablehlo/build_tools and stablehlo/build_tools/github_actions scripts have control options such as 
```
set -e

## or

set -o errexit  # Ensures the script exits immediately if any command returns a non-zero exit status
set -o nounset   # Treats the use of uninitialized variables as an error and causes the script to exit
set -o pipefail   # Ensures the script detects errors in any part of a pipeline.

```
Do we need to add them to this script too?

Reply:
done!